### PR TITLE
Origin/orderdetail

### PR DESCRIPTION
**Pull request by kelaiya**
setting up order detail model

### CODE REVIEW

File: server/db/models/orderProduct.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const Sequelize = require('sequelize')
+const db = require('../db');
+
+const OrderProduct = db.define('orderProduct',{
+	quantity: {
+		type: Sequelize.INTEGER,
+		where:{
```
Comment (sarahdherr) [inline server/db/models/orderProduct.js]: I would not include a where query in your model definition. If you want to make sure something is not less than 0, I would look into validators with min and max.

**Review by sarahdherr — COMMENTED** [review]
There is a merge conflict. I have made some comments. Once you address both these things I will be ready to merge to master. Also, there is some commented out code at the bottom of server/db/models/orderProduct.js, if you do not need this code I would delete it. If you want it for later keep it in.

File: server/db/models/orderProduct.js (hunk 1 above)
Comment (kelaiya) [reply server/db/models/orderProduct.js]: Okay...

**Comment by monicachoe** [comment]
[Setting up Order Look-up [detailed order]](https://trello.com/c/XKOwnmKC/136-setting-up-order-look-up-detailed-order)

**Pull request merged**